58437: Add entry for new CPU temperature graphs. (#757)
freenas/freenas-docs: userguide/reporting.rst

  * `CPU <https://collectd.org/wiki/index.php/Plugin:CPU>`__
    shows the amount of time spent by the CPU in various states
    such as executing user code, executing system code, and being

>     idle. There are also CPU load graphs for short, mid, and long terms

Reviewer: These should be hyphenated, and the "There are" is a little awkward.
```suggestion
    idle. Graphs of short-, mid-, and long-term load are shown, along with
```

>     and CPU temperature graphs.

Reviewer: ```suggestion
    CPU temperature graphs.
```
